CHANGELOG.md

```diff
@@ -27,12 +27,20 @@ Release built: _not released yet_
 >   - `/state/entity/page/non-fungibles/` (when using `non_fungible_include_nfids` opt-in)
 >   - `/state/entity/page/non-fungible-vaults/` (when using `non_fungible_include_nfids` opt-in)
 
+### What’s new?
+- New configuration options `DataAggregator__Storage__StoreTransactionReceiptEvents`, and `DataAggregator__Storage__StoreReceiptStateUpdates` for data aggregator to configure if transaction's receipt events and receipt state updates should be stored in the database. It is meant to be used by gateway runners who want to reduce their database size. Keep in mind that when disabled `/stream/transactions` and `/stream/transactions` will not return that data.
```

Comment:
A few things:
* `/stream/transactions` is repeated twice - do you mean a different endpoint?
* On first reading, it sounds like `/stream/transactions` won't work? Do you mean the receipt fields for the given transactions return `null` or `{}` or is the field just missing? or something? 

I think we want:
```suggestion
- New configuration options `DataAggregator__Storage__StoreTransactionReceiptEvents`, and `DataAggregator__Storage__StoreReceiptStateUpdates` for the data aggregator to configure if a transaction's receipt events and receipt state updates should be stored in the database. It is meant to be used by gateway runners who want to reduce their database size. Keep in mind that when disabled, the corresponding properties will be missing on the `/stream/transactions` response for filtered out transactions. You can save significant space by using `StoryOnlyForUserTransactionsAndEpochChanges` and only excluding round change transactions, which aren't typically read from the `/stream/transactions` endpoint.
```

Reply:
ok committed slightly altered suggestion (it also affects committed-details endpoint).